CHANGELOG.md

```diff
@@ -6,9 +6,19 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+## [0.18.1] - 2019-07-19
+
 ### Changed
 
 - ([GH-530](https://github.com/lingua-pupuli/puppet-vscode/issues/530)) Enable extension with Puppet workspace files
+- ([GH-141](https://github.com/lingua-pupuli/puppet-editor-services/issues/141)) [puppet-editor-services-0.20.0](https://github.com/lingua-pupuli/puppet-editor-services/commit/31b5c35ea6e1b3d2dcabcf64d4a9eeffbdf0dc61) Load all Puppet 4 API Features
```

Comment:
Would a better link be the release tag?

[https://github.com/lingua-pupuli/puppet-editor-services/releases/tag/0.20.0](https://github.com/lingua-pupuli/puppet-editor-services/releases/tag/0.20.0)